### PR TITLE
Add 'getrpcinfo' missing method for version 0.18.0

### PR DESCRIPTION
**Pull request by sskender**

### CODE REVIEW

**Comment by pedrobranco** [comment]
Can you include in this PR all the other [0.18 methods](https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.18.0.md#new-rpcs)?

**Comment by pedrobranco** [comment]
And also add the [deprecated RPCs](https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.18.0.md#deprecated-or-removed-rpcs).

**Comment by sskender** [comment]
Included new and deprecated 0.18 RPC methods

File: src/methods.js [hunk 1]
```diff
@@ -179,7 +179,7 @@ export default {
     features: {
       multiwallet: '>=0.15.0'
     },
-    version: '>=0.11.0'
```
Comment (pedrobranco) [inline src/methods.js]: Sorry about the misinformation in [here](https://github.com/ruimarinho/bitcoin-core/pull/100#issuecomment-532198709) but revert this change because `generate` is only deprecated (not removed).
